contracts/gas-service/AxelarGasService.sol

```diff
@@ -4,7 +4,7 @@ pragma solidity 0.8.9;
 
```

Comment:
Please don't change gateway and then gas service in the same PR in the future. PRs need to be focused in terms of the context.